modules/contacts/modules.php

```diff
@@ -6,328 +6,510 @@
  * @subpackage contacts
  */
 
-if (!defined('DEBUG_MODE')) { die(); }
+if (!defined('DEBUG_MODE')) {
+	die();
+}
 
-require APP_PATH.'modules/contacts/hm-contacts.php';
+require APP_PATH . 'modules/contacts/hm-contacts.php';
 
 /**
  * @subpackage contacts/handler
  */
-class Hm_Handler_autocomplete_contact extends Hm_Handler_Module {
-    public function process() {
-        list($success, $form) = $this->process_form(array('contact_value'));
-        $results = array();
-        if ($success) {
-            $val = trim($form['contact_value']);
-            $contacts = $this->get('contact_store');
-            $contacts->sort('email_address');
-            $results = array_slice($contacts->search(array(
-                'display_name' => $val,
-                'email_address' => $val
-            )), 0, 10);
-        }
-        $this->out('contact_suggestions', $results);
-    }
+class Hm_Handler_process_contact_auto_collect_setting extends Hm_Handler_Module
```

Comment:
Please revert all these syntax changes - I think you used an IDE to do it but this has negative side effects:
- makes it rather hard to review the actual code that changed
- introduces code changes in git history that pollute the history and don't improve anything

I'd like to see a diff of the actual changes you did here, so please either do them again on a branch starting from fresh master or revert the syntax changes by your IDE.

Reply:
Alright, please let me know when you leave only the actual changes in this PR and I will review it again.

Reply:
Indentation is Good now from: [https://github.com/cypht-org/cypht/commit/9a14becd4ed6616e319fa8e744e4b333ed6a654a](https://github.com/cypht-org/cypht/commit/9a14becd4ed6616e319fa8e744e4b333ed6a654a )

Reply:
It is not only the indentation. There are hundreds of places where opening/closing brackets were moved to a new line. See diff here: https://github.com/cypht-org/cypht/pull/816/files

Reply:
Oh you are right, I deleted the branch and recreated it.

Reply:
Thank you!